src/pywink/devices/standard/__init__.py

```diff
@@ -129,13 +129,25 @@ def update_state(self, require_desired_state_fulfilled=False):
             if not is_desired_state_reached(power_strip[0]):
                 return
 
+    def _update_state_from_response(self, response_json, require_desired_state_fulfilled=False):
+        """
+        :param response_json: the json obj returned from query
+        :return:
+        """
+        if 'data' in response_json:
+            power_strip = response_json.get('data')
+        else:
+            power_strip = response_json
         power_strip_reading = power_strip.get('last_reading')
         outlets = power_strip.get('outlets', power_strip)
         for outlet in outlets:
             if outlet.get('outlet_id') == str(self.device_id()):
                 outlet['last_reading']['connection'] = power_strip_reading.get('connection')
                 self.json_state = outlet
 
+    def pubnub_update(self, json_response):
+        self._update_state_from_response(json_response)
```

Comment:
I would prefer to see you "sanitize" the pubnub json response *here*, converting it to a format that is consistent with the output from the Wink API.